Yoshi/QAKit/README.md

```diff
@@ -0,0 +1,48 @@
+# QAKit
+
+QAKit provides menus for some frequestly used QA tasks.
+
+## EnvironmentMenu
+
+Environment Menu is used to globally change and store the networking environment (http base URL) of your app. Yoshi leverages the new `JSONDecoder` and `JSONEnCoder` in Swift 4 to help persist the environment selection in `NSUSerDefaults`. So when user selects the menu, the selection is persisted and will be retrieved automatically next time app is opened.
```

Comment:
should be `Decoder`, `Encoder`, and `UserDefaults`

Reply:
I think `Encoder` and `Decoder` are there all the time, the new thing is `JSONDecoder` and `JSONEncoder`, thought might be good to let the user know what's being stored.